.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"java.configuration.updateBuildConfiguration": "interactive"
+}
```

Comment:
Are you sure you wanted to add these files to version control?

Reply:
No, they are useless might have been installed by android studio.